### PR TITLE
More clean up, Use groovy syntax

### PR DESCRIPTION
**Pull request by jaredsburrows**

### CODE REVIEW

File: src/test/groovy/com/vanniktech/android/apk/size/ApkSizeToolsSpec.groovy [hunk 1]
```diff
@@ -1,42 +1,45 @@
...
+    def defaultLocale = Locale.default
 
     def 'cleanup'() {
-        Locale.setDefault(defaultLocale)
+        Locale.setDefault defaultLocale
     }
 
-    def 'convert bytes to mega bytes'() {
+    @Unroll
+    def 'convert bytes to mega bytes with #locale locale'() {
```
Comment (jaredsburrows) [inline src/test/groovy/com/vanniktech/android/apk/size/ApkSizeToolsSpec.groovy]: Unroll will display each test case individually. Try running these tests in your IDE.

File: src/test/groovy/com/vanniktech/android/apk/size/ApkSizePluginSpec.groovy [hunk 2]
```diff
@@ -13,17 +13,13 @@ final class ApkSizePluginSpec extends Specification {
         project = ProjectBuilder.builder().build()
     }
 
-    @Unroll
-    def '#theProject project is invalid'() {
+    def 'project is invalid'() {
```
Comment (jaredsburrows) [inline src/test/groovy/com/vanniktech/android/apk/size/ApkSizePluginSpec.groovy:15]: No need for `null`.

File: build.gradle [hunk 3]
```diff
@@ -50,22 +48,22 @@ pluginBundle {
...
 task wrapper(type: Wrapper) {
     gradleVersion = '3.3'
-    distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
+    distributionType = Wrapper.DistributionType.ALL
 }
 
 task groovydocJar(type: Jar, dependsOn: groovydoc) {
     classifier = 'groovydoc'
-    from 'build/docs/groovydoc'
+    from groovydoc.destinationDir
```
Comment (jaredsburrows) [inline build.gradle:56]: Use of constants.

**Comment by jaredsburrows** [comment]
@vanniktech This is confusing. Locally, Jacoco says 51%. Using the "coverage" plugin in Intellij, it says 64%.

File: build.gradle [hunk 4]
```diff
@@ -50,22 +48,22 @@ pluginBundle {
 
 task wrapper(type: Wrapper) {
     gradleVersion = '3.3'
-    distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
+    distributionType = Wrapper.DistributionType.ALL
```
Comment (vanniktech) [inline build.gradle]: that's neat

Comment (vanniktech) [inline build.gradle:51]: that's neat

File: src/main/groovy/com/vanniktech/android/apk/size/ApkSizeTask.groovy [hunk 5]
```diff
@@ -1,25 +1,24 @@
...
-
-import groovy.transform.stc.ClosureParams
-import groovy.transform.stc.SimpleType
 
 class ApkSizeTask extends DefaultTask {
-    @Input def File apk
-    @OutputFile def File outputFile
+    @Input
+    File apk
+    @OutputFile
```
Comment (vanniktech) [inline src/main/groovy/com/vanniktech/android/apk/size/ApkSizeTask.groovy]: is it possible to have those on the same line? (just as it was before)

**Comment by jaredsburrows** [comment]
@vanniktech All fixed.

**Review by vanniktech — APPROVED** [review]
Thanks 👍 yeah I know coverage always seems different and I also would not trust the number that much

**Comment by jaredsburrows** [comment]
@vanniktech Could we switch to using coveralls?

**Pull request merged**